mteb/abstasks/TaskMetadata.py

```diff
@@ -144,3 +146,6 @@ class TaskMetadata(BaseModel):
 
     text_creation: TEXT_CREATION_METHOD | None
     bibtex_citation: str | None
+
+    n_samples: dict[str, int]
+    avg_character_length: dict[str, float]
```

Comment:
We've made the fields nullable for the other ones (instead of an empty container, e.g. `{}`). Perhaps we should do that here as well?

Reply:
Fixed